### PR TITLE
Organize Folders, issue #116

### LINKED ISSUE #116: Folder Structure
## Problem
Right now, the folder structure is inconsistent. Some components have their own folders, and some don't.

image

## Goal 
Each component and its stylesheet in `/src/components` should have its own folder. For example, Actions.js and Actions.scss should be in the folder `Actions`. File paths and imports should also be updated to reflect these changes.

### PR DESCRIPTION
**Pull request by arniber21**
Fixes error #116 with the folder structures

Went through the webpage before, through the challenger page and through the basic workflow. After going through and organizing folders, everything worked as usual. 

I made a folder called "utility" to contain straggling components that didn't fit into their own directory. I also left the App.js and App.scss by itself w/o a folder as that is common practice across React.

### CODE REVIEW

**Review by Devanshusp — DISMISSED** [review]
Hey @arniber21, thanks for making so many organizational changes, it was definitely due.

I like how you separated the folders; the only changes I would make are moving the `VoterRegistration` folder and the `ElectionReminders` folder inside the `Actions` folder. Also, the `Notification` folder inside the `Utility` folder. However, this is not required, it still makes sense the way you have it organized.

Otherwise, everything else looks great! There are a few merge conflicts, I'll merge once those are cleared.

**Comment by arniber21** [comment]
Should be all fixed now, let me know if you get any errors. It was smooth sailing for me at my end

**Comment by arniber21** [comment]
Had to fix a merge conflict real quick - should be good now

**Comment by arniber21** [comment]
Fixed merge issues

File: src/components/Actions/Actions.js [hunk 1]
```diff
@@ -1,84 +1,108 @@
...
+import ConfettiAnimation from "../Utility/Helpers/ConfettiAnimation";
+import Invite from "../Utility/Invite/Invite.js";
 import "./Actions.scss";
+<<<<<<< HEAD
+import {LoadingWheel} from "../Utility/LoadingWheel/LoadingWheel.component";
+=======
 import { LoadingWheel } from "../LoadingWheel/LoadingWheel.component";
+<<<<<<< HEAD
+import MetaTags from 'react-meta-tags';
+>>>>>>> 0ae281d19ec71fc1ea157a19c9ae493d20e1c524
```
Comment (Devanshusp) [inline src/components/Actions/Actions.js]: This looks like git version management
Its preventing some of the files to run

**Review by Devanshusp — CHANGES_REQUESTED** [review]
Hey Arnab, the folder organization looks great, just some merge conflicts (something weird is going on with the git).

I'll merge once those are fixed!

File: src/components/Actions/Actions.js (hunk 1 above)
Comment (arniber21) [reply src/components/Actions/Actions.js]: Huh thats weird, lemme fix that rq

**Comment by arniber21** [comment]
I went through and fixed the git errors, site works for me. Although I would check on your end.

**Review by Devanshusp — APPROVED** [review]
Thanks @arniber21! Merging now

**Pull request merged**